packages/web-app-files/src/index.ts

```diff
@@ -54,6 +54,10 @@ export const navItems = (context): AppNavigationItem[] => {
         return !spacesStores.currentSpace || spacesStores.currentSpace?.isOwner(userStore.user)
       },
       enabled() {
+        if (!spacesStores.spaces.length) {
```

Comment:
There is `spacesStores.spacesInitialized` which seems to be a better fit for what you're doing here